### PR TITLE
feat: Make invalid filename error detectable

### PR DESCRIPTION
**Pull request by sm2017**
##### Checklist

- [x] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

##### Affected core subsystem(s)

Nothing

##### Description of change

Allow to detect invalid file name error

### CODE REVIEW

**Comment by sm2017** [comment]
@dead-horse please review

**Comment by sm2017** [comment]
@dead-horse please review 
@atian25 please accept PR

File: app.js [hunk 1]
```diff
@@ -70,6 +70,7 @@ module.exports = app => {
       try {
         if (!checkExt(filename)) {
           const err = new Error('Invalid filename: ' + filename);
+          err.name = 'MultipartFileInvalidFilenameError';
```
[the remark this answers is not in the document]
Comment (sm2017) [reply app.js]: @dead-horse How? Why `err.name` must won't be changed?

[the remark this answers is not in the document]
Comment (fengmk2) [reply app.js]: `MultipartInvalidFilenameError` will be better.

**Comment by atian25** [comment]
the baseline has refactor, if you still need this feature, fell welcome to rebase it and add some tests thanks

**Pull request closed without merging**